pkg/crc/preflight/preflight_checks_windows.go

```diff
@@ -166,3 +167,41 @@ func checkIfRunningAsNormalUser() error {
 	logging.Debug("Ran as administrator")
 	return fmt.Errorf("crc should be ran as a normal user")
 }
+
+func removeDnsServerAddress() error {
+	//Set-DNSClientServerAddress "vEthernet (Default Switch)" -ResetServerAddresses
+	resetDnsForDefaultSwitchCommand := `Set-DnsClientServerAddress "vEthernet (Default Switch)" -ResetServerAddresses`
```

Comment:
Maybe combine this in a single command to execute, so we don't show two UAC popups.

Reply:
@gbraad I also thought same but may be https://docs.microsoft.com/en-us/powershell/module/dnsclient/set-dnsclientserveraddress?view=win10-ps#syntax `InterfaceAlias` might work, will need to check.

Reply:
you likely just need to add both commands with a separator. the Alias is an alternative name for the network interface, which "crc" and "default switch" actually are.

Reply:
@gbraad so in this case alias is much better than adding the commands with separator right?

Reply:
I am not sure what you mean:

```
PS> Set-DnsClientServerAddress "crc" -ResetServerAddresses`
```

is the same as:

```
PS> Set-DnsClientServerAddress -InterfaceAlias "vEthernet (Default Switch)" -ResetServerAddresses
```

I mean to combine them to execute as:

```
PS Set-DnsClientServerAddress "crc" -ResetServerAddresses; Set-DnsClientServerAddress "default switch" -ResetServerAddresses

Reply:
@gbraad How about following?

```
PS> Set-DnsClientServerAddress -InterfaceAlias ("default switch","crc") -ResetServerAddresses
```

Reply:
that should be:


```
PS> Set-DnsClientServerAddress -InterfaceAlias ("vEthernet (Default Switch)","vEthernet (crc)") -ResetServerAddresses
```

as Alias needs the actual `vEthernet` prefix. Also note, that "Default Switch" might not be named as such on localized versions of Windows. This is already part of the code and best to use this replacement value.

Reply:
https://github.com/code-ready/crc/blob/20d32e725098acf94d9032f0799e43224b9b82f5/pkg/os/windows/network/vswitch_windows.go#L37-L47

Reply:
which would return nothing if the default does not exist. and could prevent the second device...

but it would still need to be collected in a single statement to prevent the UAC popup IF we by accident have configured it once (before they started to use the one named crc)

Reply:
@gbraad How about following, this way we delete the both and single time UAC popup?

```
func removeDnsServerAddress() error {
	resetDnsForDefaultSwitchCommand := `Set-DnsClientServerAddress -InterfaceAlias ("vEthernet (crc)") -ResetServerAddresses`
	if exist, defaultSwitch := winnet.GetDefaultSwitchName(); exist {
		resetDnsForDefaultSwitchCommand = fmt.Sprintf(`Set-DnsClientServerAddress -InterfaceAlias ("vEthernet (%s)","vEthernet (crc)") -ResetServerAddresses`, defaultSwitch)
	}
	if _, _, err := powershell.ExecuteAsAdmin("Remove dns entry for default switch", resetDnsForDefaultSwitchCommand); err != nil {
		return err
	}
	return nil
}
```